src/style-toaster.scss

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+
+// Support for auroElement styles
+// @import './node_modules/@aurodesignsystem/webcorestylesheets/dist/auroElement/auroElement';
```

Comment:
should this be commented out?

Reply:
i _think_ it's ok commented out for now. it's commented out because that's how the template of this repo had it. @blackfalcon, is this still ok to be commented out?